ChemistryLib/PhreeqcIOData/Surface.h

```diff
@@ -0,0 +1,46 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2019, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#pragma once
+
+#include <iosfwd>
+#include <string>
+
+namespace BaseLib
+{
+class ConfigTree;
+}
+
+namespace ChemistryLib
+{
+namespace PhreeqcIOData
+{
+struct SurfaceSite
+{
+    SurfaceSite(std::string&& mineral_,
+                double const site_density_,
```

Comment:
please add a variable for the flag "-sites".
In Phreeqc you can specifiy a surface site by giving the *-site_density* in n/m2 or by giving the absolute amount of surface sites by *-sites*.

Reply:
Good point. I will follow your suggestion to implement further extension.

Reply:
I would also suggest to stick to the Phreeqc flag names: `mineral` --> `name`.

Reply:
> I would also suggest to stick to the Phreeqc flag names: `mineral` --> `name`.

Changed.